### DISCUSSION TITLE
RDMA error -734009

### DISCUSSION THREAD

**Discussion by WillsHuang2022** [ni/easyrdma#14] [Q&A]
![image](https://user-images.githubusercontent.com/109494055/219545081-d77fdb06-18d3-43ca-831d-3f6217bb3236.png)

Currently, we meet RDMA -734009 at our project accidentally, and I cannot search for the detail error description, can you please give us some support what is the explanation for this error, and what suggestion to solve this error?

**Comment by nikhalim** [comment]
It seems that the code ran into an error with renaming the thread that it's trying to create. Can you provide more information on how this error was encountered? If you can share the setup, how RDMA API is being called, etc, that would be great.

**Reply by nikhalim** [reply]
@WillsHuang2022 Are you still having this issue?

**Comment by nikhalim** [comment]
Closing this discussion since there hasn't been any response from author
